sockeye/output_handler.py

```diff
@@ -119,6 +121,54 @@ def handle(self,
         self.stream.flush()
 
 
+class NematusStringWithAlignmentsOutputHandler(StringOutputHandler):
+    """
+    Output handler to write translations and alignments to a stream. The format is compatible with the output
+    generated by Nematus. Note that unlike other output handlers each input sentence will result in an output
```

Comment:
Lets not use "Nematus" anywhere because its not really Nematus-related but rather compatible to another tool.
The docstring here could mention the link to the SoftAlignments project.
So maybe "SoftAlignmentsVisualizerOutputHandler"

Reply:
Yeah, it's just the format used by nematus, but I completely agree that it would make sense to leave "nematus" out as it is just a different output format that happens to be compatible with nematus. mhh on the other hand it's also not SoftAlignments specific. I was also considering "translation_with_details", but I'm also not super happy about that. "translation_with_alignment_matrix" and StringWithAlignmentMatrixOutputHandler, how's that?

Reply:
Sounds good!